PHPCompatibility/Sniffs/PHP/ArgumentFunctionsReportCurrentValueSniff.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * \PHPCompatibility\Sniffs\PHP\ArgumentFunctionsReportCurrentValue.
+ *
+ * PHP version 7.0
+ *
+ * @category PHP
+ * @package  PHPCompatibility
+ * @author   Wim Godden <wim.godden@cu.be>
+ */
+
+namespace PHPCompatibility\Sniffs\PHP;
+
+use PHPCompatibility\Sniff;
+
+/**
+ * \PHPCompatibility\Sniffs\PHP\ArgumentFunctionsReportCurrentValue.
+ *
+ * Functions inspecting function arguments report the current value instead of the original since PHP 7.0.
+ *
+ * PHP version 7.0
+ *
+ * @category PHP
+ * @package  PHPCompatibility
+ * @author   Wim Godden <wim.godden@cu.be>
+ */
+class ArgumentFunctionsReportCurrentValueSniff extends Sniff
+{
+    /**
+     * A list of functions that, when called, have a different behaviour in PHP 7 when dealing with parameters of the function they're called in.
+     * @var array(string)
+     */
+    
+    protected $changedFunctions = array(
+        'func_get_arg',
```

Comment:
If you change the format of the array to:
```php
'func_get_arg' => true,
```
you can use the more efficient `isset()` to check the array instead of `in_array()`.